src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -3417,188 +3430,189 @@ void Fishing_UpdateFish(Actor* thisx, PlayState* play2) {
+                    sLureBitTimer = 30;
+                    sRumbleDelay = 40;
                     Rumble_Override(0.0f, 180, 90, 10);
                 }
 
-                D_80B7E124 = 0;
-                D_80B7E116 = 100;
-                D_80B7E080 = 0;
+                sLineHooked = false;
+                sRodPullback = 100;
+                sFishFightTime = 0;
             }
             break;
 
         case 5:
             this->actor.uncullZoneForward = 1200.0f;
             this->actor.uncullZoneScale = 200.0f;
 
-            D_80B7E080++;
-            osSyncPrintf("HIT FISH %dcm\n", (u8)this->unk_1AC);
+            sFishFightTime++;
+            osSyncPrintf("HIT FISH %dcm\n", (u8)this->fishLength);
 
-            Math_ApproachS(&this->unk_170, 0x2AF8, 4, 0xBB8);
+            Math_ApproachS(&this->fishLimbDRotZDelta, 0x2AF8, 4, 0xBB8);
             sFishingHookedFish = this;
             Math_ApproachS(&player->actor.shape.rot.y, this->actor.yawTowardsPlayer + 0x8000, 5, 0x500);
 
-            if (D_80B7E124 == 0) {
-                if ((D_80B7FEA0 < 20) && ((D_80B7E0AE & 3) == 0)) {
-                    D_80B7FEA0++;
+            if (sLineHooked == 0) {
+                if ((sSinkingLureSegmentIndex < 20) && ((sLureTimer & 3) == 0)) {
+                    sSinkingLureSegmentIndex++;
                 }
             }
 
-            if ((D_80B7E122 != 0) && (D_80B7E124 == 0)) {
-                if (((input->rel.stick_y < -50) && (D_80B7A6C8 > -40)) || CHECK_BTN_ALL(input->press.button, BTN_A)) {
+            if ((sLureBitTimer != 0) && (sLineHooked == 0)) { // pull the line to hook it
+                if (((input->rel.stick_y < -50) && (sStickAdjYPrev > -40)) ||
+                    CHECK_BTN_ALL(input->press.button, BTN_A)) {
                     if (input->rel.stick_y < -50) {
-                        temp_f0 = 40.0f - ((this->unk_1AC - 30.0f) * 1.333333f);
+                        temp_f0 = 40.0f - ((this->fishLength - 30.0f) * 1.333333f);
                         if (temp_f0 > 0.0f) {
                             this->unk_152 = temp_f0;
                             this->unk_154 = this->actor.yawTowardsPlayer - this->actor.shape.rot.y;
                             this->unk_156 = 1;
                         }
                     }
 
-                    this->unk_198 = 1.7f;
-                    this->unk_19C = 7000.0f;
-                    D_80B7E124 = 1;
+                    this->fishLimbRotPhaseStep = 1.7f;
+                    this->fishLimbRotPhaseMag = 7000.0f;
+                    sLineHooked = 1;
                     SEQCMD_PLAY_SEQUENCE(SEQ_PLAYER_BGM_MAIN, 0, 8, NA_BGM_ENEMY);
-                    D_80B7E0A6 = 0;
+                    sFishingMusicDelay = 0;
 
-                    if (this->unk_150 == 1) {
-                        spA4 = (this->unk_1AC * 3.0f) + 120.0f;
+                    if (this->isLoach == 1) {
+                        rumbleStr = (this->fishLength * 3.0f) + 120.0f;
```

Comment:
Personally, I think this would probably be better as `rumbleStrength`. `Str` just makes me think of strings.